### PR TITLE
Forbid creating graphite collections with duplicated params

### PR DESCRIPTION
**Pull request by v0devil**

### CODE REVIEW

**Comment by kofrezo** [comment]
Can we add this to the Template, Variation and Numeric params input as well ?

File: serveradmin/graphite/admin.py [hunk 1]
```diff
@@ -38,4 +38,16 @@ class CollectionAdmin(admin.ModelAdmin):
     list_filter = ['overview']
 
 
-admin.site.register(Collection, CollectionAdmin)
+class CollectionAdminForm(forms.ModelForm):
+
+    def clean_params(self):
```
Comment (kofrezo) [inline serveradmin/graphite/admin.py:43]: Can we add this to the Template, Variation and Numeric params input as well ?

Comment (kofrezo) [reply serveradmin/graphite/admin.py:69]: I am getting an `AttributeError` exception when submitting a parameter value for Numeric or Variation now:

```
'NumericFormFormSet' object has no attribute 'cleaned_data'
```

```
'VariationFormFormSet' object has no attribute 'cleaned_data'
```

Comment (v0devil) [reply serveradmin/graphite/admin.py:69]: Ok, it cuz other form field were invalid

**Pull request merged**